pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display updated information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display a per-process summary:
```

Comment:
Does `display` sounds like only show once and won't refresh?

This command will monitor processes,  display a per-process summary and refresh result periodically.

Reply:
No, print does though.

Reply:
OK, I understand, thanks :)